src/libraries/System.Private.CoreLib/src/System/Reflection/AssemblyName.cs

```diff
@@ -172,7 +194,7 @@ public static AssemblyName GetAssemblyName(string assemblyFile)
             if (assemblyFile == null)
```

Comment:
Move this null check in S.R.Metadata part?

Reply:
Good point, since we want to make S.R.Metadata entry point accessible on its own.